program/FORCE-RISCV ISG preliminary project proposal.md

```diff
@@ -0,0 +1,145 @@
+# OpenHW Preliminary Project Proposal
+# FORCE-RISCV ISG
+
+
+## Summary of project
+FORCE-RISCV ISG has been developed in Futurewei Technologies and delivered to OpenHW Group on June 12 2020.
+
+FORCE-RISCV is a powerful verification tool for RISC-V based CPU design,  currently supports 64-bit version of RISC-V with RV64I, M, A, Zicsr, F, D, C extensions, Machine, Supervisor, User modes, dynamic ISS interaction, dynamic Virtual Memory management, loops, configurable state transition, powerful and extensible Python scriptable test template writing framework etc.  Vector extension 0.9 support, full paging exception control and advanced memory sub-system verification features are in the work and nearly finished.
+
+Note that support for 32-bit RISC-V is not there yet, but it will be a major part of this project proposal and will be supported very soon.  With 32-bit RISC-V support, many of the current OpenHW Group member's verification need will be satisfied with the powerful features provided by FORCE-RISCV.
+
+Additional features like MP/MT support and advanced resource dependency is also on the near term plan.
+
+### Summary of Timeline
+- Preliminary Project Launch (PPL) in the OpenHW Group process is proposed for end September 2020
+- Project Launch (PL) in the OpenHW Group process is proposed for October 2020.
+- Project Plan Available (PPA) is proposed for mid-October 2020
+- 32-bit version support should be useable by end-of-year

```

Comment:
Let's target Oct TWG for both PL and PPA - I think the project plan it should be relatively straightforward.

Reply:
Sounds good.